### PR TITLE
Fix Audit log, change configuration for local execution

### PR DESCRIPTION
**Pull request by BurnedMarshal**
Audit log for `GetBPDTransactions` has now the right `OperationName`
Refactor configuration for local execution.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "build": "tsc",
     "watch": "tsc -w",
     "extensions:install": "func extensions install",
-    "start": "dotenv -e .env func start --javascript",
+    "start": "dotenv -e .env.prod func start --javascript",
```
Comment (AleDore) [inline package.json:13]: Why `.env.prod` instead of `.env` for local development?

Comment (BurnedMarshal) [reply package.json:13]: `.env` is used by `docker-compose` running, `.env.prod` is used to run the backend with production/stage configuration for database and Azure storage services

Comment (BurnedMarshal) [reply package.json:13]: We could use anyway the same `.env` file, changing the values if you think that is more clear.

Comment (BurnedMarshal) [reply package.json]: Fixen in 026bdb2

**Review by AleDore — APPROVED** [review]
LGTM

**Pull request merged**